src/app/core/component/markdown-content-container/markdown-pattern-sectioncontent/markdown-pattern-section-content.component.ts

```diff
@@ -69,11 +69,15 @@ export class MarkdownPatternSectionContentComponent extends DataRenderingCompone
     this.changeText(this.renderedData);
   }
 
-  changeText(value: string): void {
+  changeText(value: string | undefined): void {
```

Comment:
Try out i this also works:
```suggestion
  changeText(value?: string): void {
```